docs/extensions/mb-custom-table.md

```diff
@@ -572,20 +572,24 @@ These filters should return an array of data (e.g., the `$row`).
 
 ### Bulk actions handling
 
-The plugin provides a way to handle bulk actions for custom models. By default, it already supports the `Delete` action. You can add your own custom bulk actions handler by creating a function following the naming convention `mbct_{$action}_bulk_action`. Please note that the action name are auto convert to lowercase and use underscores instead of hyphens in order to match with PHP function.
+The plugin provides a way to handle bulk actions for custom models. By default, it already supports the `Delete` action. 
 
-For example, this code adds a new a custom bulk action `my_action` with a handler:
+To add more custom bulk actions, you can use the `mbct_{$model}_bulk_actions` filter. This filter accepts an array of bulk actions. The key is the action name, and the value is the action label.
 
 ```php
 add_filter( 'mbct_transaction_bulk_actions', function ( $actions ) {
 	$actions['my-action'] = 'My Action';
 	
 	return $actions;
 } );
+```
+
+You can add your own custom bulk actions handler by creating a function following the naming convention `mbct_{$action}_bulk_action`. Please note that the action name are auto convert to lowercase and use underscores instead of hyphens in order to match with PHP function. In this case, `my-action` will be `my_action`.
 
+```php
 function mbct_my_action_bulk_action( $request ) {
 	// Do something with the request
-	print_r($request);
+	print_r( $request );
```

Comment:
A nghĩ nên bổ sung thêm VD về lấy thông tin qua `$request` này. VD như `$request->get( $param )` và `$request->post( $param )`

Reply:
Ok a, e update thành bài toán thực tế luôn, match với các ví dụ bên trên, user copy paste là chạy.

Ngoài ra e thấy `$ids` và `$model` chắc chắn sẽ thường dùng đấy a, vì bulk action là luôn cần id với $model mà. Mình có nên cho nó vào param thứ 2 cho người dùng tiện hơn không, như vậy thì không phải viết đoạn code get với check id mỗi lần gọi hàm đó nữa.

Như đoạn này có thể lược bỏ:
```php
$ids        = array_map( 'absint', $request->post( 'ids' ) );
$model_name = sanitize_text_field( $request->post( 'model' ) );
$model      = MetaBox\CustomTable\Model\Factory::get( $model_name );

if ( ! $ids || ! $model_name || ! $model ) {
    wp_send_json_error( __( 'Invalid request', 'mb-custom-table' ) );
}
```

Mình có thể prepare trước đó rồi khi người dùng viết:

```php
function mbct_refund_all_bulk_action($request, $atts) {
    [$ids, $model] = $atts;
   
    foreach ( $ids as $id ) {
     \MetaBox\CustomTable\API::update( $id, $model->table, [
	    'status' => 'refunded',
     ] );
    }

    wp_send_json_success();
}
```

Reply:
Nếu thế thì truyền thêm luôn 2 tham số là `$ids` và `$model` luôn e ạ.